app/controllers/api/users_controller.rb

```diff
@@ -26,5 +26,21 @@ def login
         }
       }.to_json
     end
+
+    def show
+      user = User.find_by(id: params[:id])
+      if user.nil?
+        render json: {
+          errors: 'No user found'
+        }, status: :not_found
+      else
+        render json: {
+          user: {
+            name: user.name,
+            scores: user.scores.order(played_at: :desc, id: :desc)
```

Comment:
When responding to a request, we should be careful about the data we are sending. For each request, we should always (if not specified otherwise) send the same attributes.

In this case, `user.scores.order(played_at: :desc, id: :desc)` is going to return the scores with all of their attributes. At this moment, this is fine, but if we add other fields to a score, we may not want to send them through this API. 

A safer way is to use a **serializer** - basically a function that takes a score and selects only the needed attributes. You can either create a new serializer or use the already existing one from [score.rb#serializer](https://github.com/cristian-scarisoreanu/golfr_backend/blob/main/app/models/score.rb#L10). Take as example of usage [scores_controller.rb#user_feed](https://github.com/cristian-scarisoreanu/golfr_backend/blob/main/app/controllers/api/scores_controller.rb#L7).